addons/mass_mailing/static/src/js/mass_mailing_wysiwyg.js

```diff
@@ -13,6 +11,8 @@ export class MassMailingWysiwyg extends Wysiwyg {
     //--------------------------------------------------------------------------
 
     async startEdition() {
+        this.floatingToolbarEl = this.toolbarRef.el.firstChild;
```

Comment:
1. As this toolbar is not always floating, could we name it something like`mainToolbarEl` ?
2. Could we add a comment as differentiating between `mainToolbar` and `snippetsMenuToolbarEl` is not straightforward?
```
        // The initial toolbar of the Wysiwyg component will be used as the
        // mainToolbar (sticky on mobile and floating on desktop).
```